apps/web-connect/src/features/checkout/components/PurchaseSuccessful.tsx

```diff
@@ -19,7 +19,6 @@ const PurchaseSuccessful: React.FC<IPurchaseSuccessful> = ({ returnToClient }) =
 	}
 
 	const handleReturnToClient = () => {
-        window.location = `xai-sentry://purchase-successful?txHash=${getHash()}` as unknown as Location;
```

Comment:
Is this the function of the button "Return to client" ?

If yes, should this not still open the desktop app ?

Reply:
Per the ticket, it should not [ticket](https://www.pivotaltracker.com/n/projects/2671427/stories/188028388)

Reply:
Perfect, sorry about the confusion